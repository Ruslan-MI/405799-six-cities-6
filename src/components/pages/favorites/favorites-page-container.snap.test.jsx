import React from "react";
import {
  render,
} from "@testing-library/react";
import FavoritesPageContainer from "./favorites-page-container";
import {
  StoreNameSpace,
} from "../../../const";
import Test, {
  mockDataInitialState,
  mockUserInitialState,
} from "../../../mocks-for-test";

it(`Favorites page container should render correctly`, () => {
  const dataForStore = {
    [StoreNameSpace.DATA]: mockDataInitialState,
    [StoreNameSpace.USER]: mockUserInitialState,
  };

  const {
    container,
  } = render(<Test dataForStore={dataForStore}>
    <FavoritesPageContainer />
  </Test>);

  expect(container).toMatchSnapshot();
});
