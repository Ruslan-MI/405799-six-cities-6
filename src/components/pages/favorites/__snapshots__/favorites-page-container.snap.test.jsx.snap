// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites page container should render correctly 1`] = `
<div>
  <p
    style="color: rgb(68, 129, 195); text-align: center; font-weight: bolder; font-size: 50px; padding-top: 150px;"
  >
    Loading...
  </p>
</div>
`;
